### PR TITLE
improve wording

### PR DESCRIPTION
**Pull request by delgod**

### CODE REVIEW

File: lib/charms/data_platform_libs/v0/database_requires.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Relation requirer side abstraction for database relation.
+"""Relation require side abstraction for database relation.
```
Comment (jnsgruk) [inline lib/charms/data_platform_libs/v0/database_requires.py:15]:
```suggestion
"""Relation 'requires' side abstraction for database relation.
```

File: lib/charms/data_platform_libs/v0/database_requires.py [hunk 2]
```diff
@@ -55,12 +55,12 @@ def _on_database_created(self, event: DatabaseCreatedEvent) -> None:
 As shown above, the library provides some custom events to handle specific situations,
 which are listed below:
 
-- database_created: event emitted when the requested database was created
-- endpoints_changed: event emitted when the read/write endpoints of the database have changed
-- read_only_endpoints_changed: event emitted when the read-only endpoints of the database
-  have changed
+— database_created: event emitted when the requested database was created.
```
Comment (jnsgruk) [inline lib/charms/data_platform_libs/v0/database_requires.py:58]:
```suggestion
— database_created: event emitted when the requested database is created.
```

File: lib/charms/data_platform_libs/v0/database_requires.py [hunk 3]
```diff
@@ -244,9 +244,10 @@ class DatabaseEvents(CharmEvents):
 Diff.__doc__ = """
 A tuple for storing the diff between two data mappings.
 
-added - keys that were added
-changed - keys that still exist but have new values
-deleted - key that were deleted"""
+— added — keys that were added.
+— changed — keys that still exist but have new values.
+— deleted — key were deleted.
```
Comment (jnsgruk) [inline lib/charms/data_platform_libs/v0/database_requires.py:249]:
```suggestion
— deleted — keys that were deleted.
```

File: lib/charms/data_platform_libs/v0/database_requires.py [hunk 4]
```diff
@@ -280,7 +281,7 @@ def __init__(
 
         # Define custom event names for each alias.
         if relations_aliases:
-            # Ensure the number of aliases match the maximum number
+            # Ensure the amount aliases match the maximum amount
```
Comment (jnsgruk) [inline lib/charms/data_platform_libs/v0/database_requires.py:284]:
```suggestion
            # Ensure the number of aliases does not exceed the maximum
```

File: lib/charms/data_platform_libs/v0/database_requires.py [hunk 5]
```diff
@@ -455,6 +456,7 @@ def _on_relation_changed_event(self, event: RelationChangedEvent) -> None:
         # (the database charm shared the credentials).
         if "username" in diff.added and "password" in diff.added:
             # Emit the default event (the one without an alias).
+            logger.info(f"database created on {datetime.now()}")
```
Comment (jnsgruk) [inline lib/charms/data_platform_libs/v0/database_requires.py:459]:
```suggestion
            logger.info("database created at %s", datetime.now())
```

**Review by jnsgruk — COMMENTED** [review]
Some nits, but overall a worthwhile change :)

**Review by jnsgruk — APPROVED** [review]

**Review by marceloneppel — APPROVED** [review]

**Pull request merged**